frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -247,12 +247,14 @@ string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text
 
 string.GitMachete.EnhancedGraphTable.automatic-discover.notification.title.cannot-discover-layout-error=Can't automatically discover repository layout
 string.GitMachete.EnhancedGraphTable.automatic-discover.task-title=Automatically discovering branch layout
+action.GitMachete.EnhancedGraphTable.automatic-discover.slide-out-skipped=Slide out skipped branches
 string.GitMachete.EnhancedGraphTable.automatic-discover.success-message=Branch layout has been automatically discovered. Edit the layout manually if needed.
 string.GitMachete.EnhancedGraphTable.empty-table-text.cannot-discover-layout=Provided machete file ({0}) is empty and branch layout can''t be automatically detected
 string.GitMachete.EnhancedGraphTable.empty-table-text.loading=Loading...
 string.GitMachete.EnhancedGraphTable.empty-table-text.only-skipped-in-machete-file=None of the branches provided by machete file ({0}) exists within the local repository
 string.GitMachete.EnhancedGraphTable.duplicated-branches-text=The following branches appear more than once in machete file:
-string.GitMachete.EnhancedGraphTable.skipped-branches-text=The following branches listed in machete file do not belong to the local repository:
+string.GitMachete.EnhancedGraphTable.skipped-branches-text=The following branches listed in machete file do not belong to the local repository: {0}
```

Comment:
Nit: I'd shorten this message somehow... coz as for now even with one branch the name is getting ellipsed away:

![image](https://user-images.githubusercontent.com/3383210/109355830-1e093900-7880-11eb-8ed8-d1663d7730d8.png)

Reply:
`Some branches listed in machete file do not exist:`?